client/plots/summary.js

```diff
@@ -356,20 +356,17 @@ export async function getPlotConfig(opts, app) {
 			sampleScatter: getDefaultScatterSettings(app)
 		},
 		mayAdjustConfig(config, edits = {}) {
+			console.log(config)
```

Comment:
@karishma-gangwani please remove this console.log

Reply:
@siosonel you mean Airen?

Reply:
@karishma-gangwani yes, sorry, I meant @airenzp

Reply:
Done now